crates/cxx-qt-build/src/lib.rs

```diff
@@ -878,22 +878,39 @@ impl CxxQtBuilder {
             for qmlcachegen_file in qml_module_registration_files.qmlcachegen {
                 cc_builder.file(qmlcachegen_file);
             }
-            // This is required, as described here: plugin_builder
+            // This is required, as described here: https://doc.qt.io/qt-6/plugins-howto.html#creating-static-plugins
             cc_builder.define("QT_STATICPLUGIN", None);
 
             // If any of the files inside the qml module change, then trigger a rerun
             for file in qml_module.qml_files {
                 println!("cargo::rerun-if-changed={}", file.path().display());
             }
 
+            // Export the .qmltypes and qmldir files into a stable path, so that tools like
+            // qmllint/qmlls can find them.
+            let plugin_dir = dir::module_export(&qml_module.uri);
```

Comment:
should this export be happening here inside the build_qml_module ? Or should this happen somehow somewhere else as part of the export phase?

Reply:
Right lets add that in another PR to keep things moving...